markdown/src/main/java/it/niedermann/android/markdown/MarkdownUtil.java

```diff
@@ -159,14 +226,21 @@ private static String runForEachCheckbox(@NonNull String markdownString, @NonNul
                 }
             }
             if (!isInFencedCodeBlock) {
-                if (lineStartsWithCheckbox(lines[i]) && lines[i].trim().length() > EListType.DASH.checkboxChecked.length()) {
```

Comment:
I wonder why I used to only consider `EListType.DASH`... This should maybe refactored to loop through the `EListType` enum.... however, nothing you will have to fix here :)